### ISSUE TITLE
Accordion Focus Styles

### ISSUE THREAD

**Issue by brandonjpierce** [gohypergiant/standard-toolkit#637]
When utilizing a menu, there are no focus states for the menu trigger when the accordion itself is keyboard focused. If you press `space` or click on the menu trigger button, the focus styles shift to a pressed like state for _only_ the menu trigger.

<img width="199" height="127" alt="Image" src="https://github.com/user-attachments/assets/fc6c1229-4e7e-4e7b-81bf-4f103261af77" />
<img width="172" height="59" alt="Image" src="https://github.com/user-attachments/assets/0cf210d8-8194-42a6-9082-adad1badb583" />
